src/components/profile/UserDetailForm/index.jsx

```diff
@@ -1,5 +1,117 @@
 import React from "react";
+import { Col } from "react-bootstrap";
+import COUNTRIES from "./countries.json";
+import Input from "../../form/SafePlaceInput";
+import "./index.scss";
 
-export default function UserDetailForm() {
-  return <div>UserDetail</div>;
+export default function UserDetail({ register }) {
+  return (
+    <Col xs={12} className="userDetailSection">
+      <div className="formInputGroup">
+        <Input
+          type="text"
+          placeholder="Your first name"
+          name="firstName"
+          ref={register()}
+          label="Your first Name"
+        />
+
+        <Input
+          type="text"
+          placeholder="Your last name"
+          name="lastName"
+          ref={register()}
+          label="Last Name"
+        />
+      </div>
+
+      <div className="formInputGroup">
+        <Input
+          type="text"
+          placeholder="Your nationality"
+          name="nationality"
+          label="Your nationality"
+          ref={register()}
+        />
+
+        <Input
+          type="tel"
+          placeholder="+90 552 477 25 15"
+          name="phoneNumber"
+          label="Phone number"
+          ref={register()}
+        />
+      </div>
+
+      <div className="formInputGroup">
+        <div className="formInput">
+          <label htmlFor="sex">
+            Sex
+            <select name="sex" id="sex" ref={register()}>
+              <option value="female">Female</option>
```

Comment:
I think you should add translation in this PR, since we are approaching to the end of development phase and we may not have much time to add translation to all pages. It would be easier if you add at least here now. We have many components without translation already

Reply:
I need to be approved and merged to add the functionality. I think I can add the translation with the functionality.